pkgs/development/python-modules/requests/default.nix

```diff
@@ -20,6 +20,8 @@ buildPythonPackage rec {
     sha256 = "sha256-J5c91KkEpPE7JjoZyGbBO5KjntHJZGVfAl8/jT11uAQ=";
   };
 
+  patches = [ ./0001-Prefer-NixOS-Nix-default-CA-bundles-over-certifi.patch ];
```

Comment:
```suggestion
  patches = [
    # Implements the where() function, usually provided by certifi, to
    # add locate the ca bundle on NixOS support NIX_SSL_CERT_FILE to
    # overwrite that logic. 
    ./0001-Prefer-NixOS-Nix-default-CA-bundles-over-certifi.patch
  ];
```

Reply:
I like the explanatory comment, but wouldn't it be better to use it as the commit message in the patch? That way if the patch itself is downloaded separately, the explanation will stay with it.

Reply:
This allows browsing the derivation and not having to look up the patch to understand what it does, that may well be my preference and not set in stone. CC @jonringer

Reply:
For now I've pushed a change that adds a detailed commit message to the patch.

Reply:
WFM.

Reply:
when reading future diffs, or modifying the code, it's nice to have the narrative co-located with the implementation.

Reply:
I agree, but by "the implementation" do you mean the patch or the Nix expression?

Reply:
I mean the nix expression, as generally I wouldn't expect most patches to contain much info (although they can)

Reply:
I see. Well, I think I'll just leave it as is for now (with detailed info in the patch file) since both of you seem to be OK with that. I actually think the patch filename visible in the .nix file already contains enough detail for people to understand generally what it's doing. Thanks for the feedback!